crypto/src/main/java/org/web3j/crypto/Sign.java

```diff
@@ -102,7 +102,7 @@ private static BigInteger recoverFromSignature(int recId, ECDSASignature sig, by
         //        routine specified in Section 2.3.7, where mlen = ⌈(log2 p)/8⌉ or mlen = ⌈m/8⌉.
         //   1.3. Convert the octet string (16 set binary digits)||X to an elliptic curve point R
         //        using the conversion routine specified in Section 2.3.4. If this conversion
-        //        routine outputs “invalid”, then do another iteration of Step 1.
+        //        routine outputs "invalid", then do another iteration of Step 1.
```

Comment:
The quotes “ and ” were causing problems with the encoding on my system, so I changed them to "standard" ASCII quotes ... happy to revert if you prefer the originals though.